tracer_test.go

```diff
@@ -255,6 +255,11 @@ func TestEmptySpanContextAsParent(t *testing.T) {
 	assert.True(t, ctx.IsValid())
 }
 
+func TestTracerClose(t *testing.T) {
+	_, tc := NewTracer("x", NewRemotelyControlledSampler("svc"), NewNullReporter())
+	assert.NoError(t, tc.Close())
```

Comment:
The simplest way is to add a test that RemoteSampler implements io.Closer(). 
A more intelligent test is to call `rSampler.pollStopped.Wait()`